ui/app/styles/app.scss

```diff
@@ -35,13 +35,13 @@ $button-shadow: 0 3px 2px rgba(var(--shadow), 0.2);
   --shadow: #{dehex(color.$ui-cool-gray-900)};
   --focus-ring: #{dehex(color.$blue-500)};
   --success: #{dehex(color.$green-050)};
-  --success-text: #{dehex(color.$green-600)};
+  --success-text: #{dehex(color.$green-700)};
   --success-border: #{dehex(color.$green-100)};
   --error: #{dehex(color.$red-050)};
   --error-text: #{dehex(color.$red-600)};
   --error-border: #{dehex(color.$red-100)};
   --warning: #{dehex(color.$yellow-050)};
-  --warning-text: #{dehex(color.$yellow-600)};
+  --warning-text: #{dehex(color.$yellow-800)};
```

Comment:
I'd be curious to see how this impacts the UI, and possibly share that with design so they're aware of good accessible defaults for the `warning` and `success` styles

Reply:
Yeah, I’d like design’s sign-off on this too. I *think* the before/after screenshots in the PR description are the extent of the changes but there may be more.

Reply:
Yeah, these look good. Orange turning to brown is always the tough one! Marketing's new stateful color palette is something we'll look to leverage in product soon 🙂